tests/nlu/test_persistor.py

```diff
@@ -165,6 +165,53 @@ def mocked_list_blobs(container_name, prefix=None):
     assert result == []
 
 
+def test_list_models_method_in_external_persistor():
+    def mocked_get_persistor(name):
+        p = Object()
+
+        def mocked_list_models():
+            return ["model_name"]
+
+        p.list_models = mocked_list_models
+        return p
+
+    with patch.object(persistor, "get_persistor", mocked_get_persistor):
+        result = persistor.get_persistor("persistor_name").list_models()
+    assert result == ["model_name"]
```

Comment:
an easy test without custom module would be to just the specify the full module path to the `aws` persistor and then check with `isinstance` whether it's the right class. This way you don't have to mock anything.